resources/constraints/version_denylist.txt

```diff
@@ -1,4 +1,3 @@
-matplotlib !=3.9.1 ; platform_system == 'Windows'
```

Comment:
Is this change intentional?

Reply:
yes. This release is yanked from PyPI

Reply:
It seems like we should have a better mechanism for that 😂 but ok!

Reply:
There is, but I prefer to validate it in another PR.